docs/dietpi_tools.md

```diff
@@ -43,17 +43,15 @@ It is one of the core tools, enabling you to install or uninstall one or more [*
 === "Software Additional"
     This section provides:
 
-    - software packages that are often used and installed via `apt install <package>`, without additional optimisation from DietPi team.
+    - software packages that are often used and installed via ```apt install <package>```, without additional optimisation from DietPi team.
```

Comment:
I haven't seen it live, but I think the issue is not the single quote code here but that the whole block is indented with four white spaces, which, in Markdown, is another syntax to make a whole block pre-/code-formated.

I suggest to indent the whole block with the white spaces only, as said in the other comment already: Two white spaces for each indentation level.